src/api/user.js

```diff
@@ -16,7 +16,8 @@ export const schema = `
     assignment(campaignId: String): Assignment
     terms: Boolean
     profileComplete(organizationId: String): Boolean
-    cacheable: Boolean
+    notifications(organizationId: String): [Assignment]
+    notifiable: Boolean
```

Comment:
cacheable was always a placeholder for notifiable, and when it depended on postgres returning() as well, it seemed like we should name it something more precise.
`notifications` is the polled api call that can return the updated assignments with the newer data.